libgalois/include/galois/graphs/PropertyFileGraph.h

```diff
@@ -352,6 +399,56 @@ class GALOIS_EXPORT PropertyFileGraph {
   const std::shared_ptr<arrow::Table>& edge_table() const {
     return rdg_.edge_table();
   }
+
+  // Pass through topology API to match PropertyGraph API
+
+  using node_iterator = GraphTopology::node_iterator;
+  using edge_iterator = GraphTopology::edge_iterator;
+  using edges_range = GraphTopology::edges_range;
+  using iterator = GraphTopology::iterator;
+  using Node = GraphTopology::Node;
+
+  // Standard container concepts
+
+  node_iterator begin() const { return topology().begin(); }
+
+  node_iterator end() const { return topology().end(); }
+
+  size_t size() const { return topology().size(); }
+
+  bool empty() const { return topology().empty(); }
+
+  uint64_t num_nodes() const { return topology().num_nodes(); }
+  uint64_t num_edges() const { return topology().num_edges(); }
+
+  /**
+   * Gets the edge range of some node.
+   *
+   * @param node node to get the edge range of
+   * @returns iterator to edges of node
+   */
+  edges_range edges(const node_iterator& node) const {
+    return topology().edges(node);
+  }
+
+  /**
+   * Gets the first edge of some node.
+   *
+   * @param node node to get the edge of
+   * @returns iterator to first edge of node
+   */
+  edge_iterator edge_begin(Node node) const {
```

Comment:
FWIW, my original intent was to keep the new graph interface relatively orthogonal with the existing uses of `edge_begin`/`edge_end` maintained for legacy/deprecation reasons.

I don't know if it that helpful to have so many different ways to do the same thing. `edge_begin` and `edge_end` were created before C++ had `for (T item : container)`. I think `edges` should suit most new use cases better.

Reply:
I killed `edge_{begin,end}` except on `PropertyGraph`.